### ISSUE TITLE
Request: Add support for the CSS property: speak

### ISSUE THREAD

**Issue by tommai78101** [stylelint/stylelint#3285]
This is a feature request to add support for the CSS property, `speak`. This rule doesn't seem to be picked up by stylelint, as it's reporting to be unknown.

Codewise, this is what it looks like:

```css
.foo {
  speak: none;
}
```
Using stylelint 9.2.0.

I'm running stylelint through VS Code extension, vscode-stylelint.

This syntax is related to a somewhat obscure syntax for voice speaking and TTS. I'm not sure if this is non-standard syntax or standard.

Here's the CSS paper:

https://www.w3.org/TR/CSS21/aural.html#propdef-speak

I just wanted it to not be flagged as "unknown". But right now, the actual result is that `speak: none;` is unknown.

```
[CSS] Unknown property, 'speak'.
```

**Comment by evilebottnawi** [comment]
Not related to `stylelint`, ref https://github.com/known-css/known-css-properties/issues/48. Thanks for issue

**Comment by vio** [comment]
`speak` property is included in [`known-css-properties`](https://github.com/known-css/known-css-properties/blob/master/data/all.json#L862).

I think the lint error is thrown by VS Code, I'm getting the same on my setup (neovim + vscode-css-languageserver) server

**Comment by evilebottnawi** [comment]
@stylelint/contributors somebody use `vs`?

**Comment by tommai78101** [comment]
I do. For work.

**Comment by tommai78101** [comment]
https://github.com/mdn/data/issues/218

Taking it to the max level.
